doc/CONTRIBUTING.md

```diff
@@ -60,8 +60,8 @@ called `npm` is installed as well.
 
 Not all code contributions will be accepted. The safest way to ensure
 that you don't waste time working on something that gets rejected is to
```

Comment:
I might change "something" to "something major".

The reason being that we're pretty fine with minor-to-medium-sized PRs (like this one!) happening with no discussion, it's more the big radical plans (major refactors etc.) that are good to talk about first.